libcxx/cmake/config-ix.cmake

```diff
@@ -118,6 +118,11 @@ elseif(ANDROID)
   set(LIBCXX_HAS_ATOMIC_LIB NO)
 else()
   check_library_exists(pthread pthread_create "" LIBCXX_HAS_PTHREAD_LIB)
-  check_library_exists(rt clock_gettime "" LIBCXX_HAS_RT_LIB)
+  check_library_exists("" clock_gettime "" LIBCXX_HAS_RT_FUNC_WITHOUT_LIB)
```

Comment:
IMO the issue here is that `check_library_exists(rt ...)` doesn't ensure that `rt` actually exists, it seems to only ensure that `clock_gettime` can be used. What happens if we use something like `check_library_exists("" "" "" LIBCXX_HAS_RT_LIB)` instead?